### DISCUSSION TITLE
Does quic still support the draft-29 implementation? How to access it?

### DISCUSSION THREAD

**Discussion by harsh4786** [libp2p/rust-libp2p#4425] [Q&A] [answered]
Im trying to listen to nodes that support the older version of quic as shown here:

```swarm.listen_on("/ip4/0.0.0.0/udp/8999/quic".parse()?)?;```

and i get this error 

<img width="657" alt="Screenshot 2023-09-02 at 2 33 50 PM" src="https://github.com/libp2p/rust-libp2p/assets/50767810/e69cdf78-b274-48a7-9622-e6bd096afad2">
 
referring to the issue here [#3151](https://github.com/libp2p/rust-libp2p/pull/3151) how to listen to a node supporting an older quic version in the latest libp2p version?

Update: If I replace `quic` with `quic-v1` it starts listening but then i don't receive any messages when I poll for events using `select!` this might be because the nodes im connecting to don't support `quic-v1`

This is a node that i'm dialing to `"/dns4/wormhole-testnet-v2-bootstrap.certus.one/udp/8999/quic/p2p/12D3KooWAkB9ynDur1Jtoa97LBUp8RXdhzS5uHgAfdTquJbrbN7i"`

**Comment by thomaseizinger** [comment, marked as answer]
Did you enable the config flag in the transport config?

**Reply by harsh4786** [reply]
Ahh the boolean flag? very sneaky! Will try it out, please don't close this issue though.

**Reply by thomaseizinger** [reply]
Just bear in mind that we will phase this out soon: https://github.com/libp2p/rust-libp2p/issues/3395

**Reply by harsh4786** [reply]
Does rust gossipsub support `join()` like as seen in go libp2p? Or `subscribe()` does exactly that behind the scenes?

**Reply by thomaseizinger** [reply]
> Does rust gossipsub support `join()` like as seen in go libp2p? Or `subscribe()` does exactly that behind the scenes?

This seems unrelated to QUIC? Can you open a new issue that describes in detail what you are looking for?

**Reply by harsh4786** [reply]
Yeah agreed
